apps/mark-scan/backend/test/auth_helpers.ts

```diff
@@ -0,0 +1,48 @@
+import { InsertedSmartCardAuthApi } from '@votingworks/auth';
+import {
+  fakeCardlessVoterUser,
+  fakeElectionManagerUser,
+  fakePollWorkerUser,
+  fakeSessionExpiresAt,
+  mockOf,
+} from '@votingworks/test-utils';
+import { CardlessVoterUser, ElectionDefinition } from '@votingworks/types';
+
+export function mockElectionManagerAuth(
+  auth: InsertedSmartCardAuthApi,
+  electionDefinition: ElectionDefinition
+): void {
+  mockOf(auth.getAuthStatus).mockImplementation(() =>
+    Promise.resolve({
+      status: 'logged_in',
+      user: fakeElectionManagerUser(electionDefinition),
+      sessionExpiresAt: fakeSessionExpiresAt(),
+    })
+  );
+}
+
+export function mockPollWorkerAuth(
+  auth: InsertedSmartCardAuthApi,
+  electionDefinition: ElectionDefinition
+): void {
+  mockOf(auth.getAuthStatus).mockImplementation(() =>
+    Promise.resolve({
+      status: 'logged_in',
+      user: fakePollWorkerUser(electionDefinition),
+      sessionExpiresAt: fakeSessionExpiresAt(),
+    })
+  );
+}
```

Comment:
These were factored out from the next commit https://github.com/votingworks/vxsuite/pull/4628/commits/73160461edba0c6a6da5e7026a52e940c721dbe6#diff-174d3fd242de548eda5dd587bdaa2bd8fad20ec94d50930f9ec62c285e3ed6daL93